modules/pinger/src/test/java/org/hawkular/component/pinger/PingerTest.java

```diff
@@ -75,39 +75,11 @@ public void testPostPinger() throws Exception {
     public void testSslPinger() throws Exception {
 
         Pinger pinger = new Pinger();
-        PingDestination destination = new PingDestination("123","https://www.perfcake.org");
+        PingDestination destination = newDestination("https://www.perfcake.org", "GET");
```

Comment:
-1, out of the present scope. These tests pinging 3rd party machines in the wild is somewhat fragile, IMO. I'd vote for mocking the responses rather than adding further servers in foreign networks.